### PR TITLE
copy labels to pod from podTemplate

### PR DESCRIPTION
**Pull request by cheyang**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mpi-operator/24)

### CODE REVIEW

File: pkg/controllers/mpi_job_controller.go [hunk 1]
```diff
@@ -485,7 +487,7 @@ func allocateGPUs(mpiJob *kubeflow.MPIJob, gpusPerNode int, done bool) (workerRe
 		if totalGPUs < gpusPerNode {
 			workerReplicas = 1
 			gpusPerWorker = totalGPUs
-		} else if totalGPUs % gpusPerNode == 0 {
+		} else if totalGPUs%gpusPerNode == 0 {
```
Comment (gaocegege) [inline pkg/controllers/mpi_job_controller.go]: Personally, I prefer the previous way.

File: pkg/controllers/mpi_job_controller.go [hunk 2]
```diff
@@ -860,11 +862,20 @@ func newLauncherRoleBinding(mpiJob *kubeflow.MPIJob) *rbacv1.RoleBinding {
 // discover the MPIJob resource that 'owns' it.
 func newWorker(mpiJob *kubeflow.MPIJob, desiredReplicas int32, gpus int) *appsv1.StatefulSet {
 	labels := map[string]string{
-		"app": mpiJob.Name + workerSuffix,
+		// "app": mpiJob.Name + workerSuffix,
```
Comment (gaocegege) [inline pkg/controllers/mpi_job_controller.go]: If we do not need the field, we could just remove it.

File: pkg/controllers/mpi_job_controller.go (hunk 1 above)
Comment (cheyang) [reply pkg/controllers/mpi_job_controller.go]: It's changed by `go fmt` not me. :)

Please check https://goreportcard.com/report/github.com/kubeflow/mpi-operator

File: pkg/controllers/mpi_job_controller.go (hunk 2 above)
Comment (cheyang) [reply pkg/controllers/mpi_job_controller.go]: Done.

File: pkg/controllers/mpi_job_controller.go (hunk 1 above)
Comment (gaocegege) [reply pkg/controllers/mpi_job_controller.go]: Oh my fault, LGTM

**Pull request merged**